tests/test_interpolation.py

```diff
@@ -347,11 +348,106 @@ def test_incremental_dict_with_interpolation() -> None:
         ({"list": ["${ref}"], "ref": "bar"}, "list.0", "bar"),
     ],
 )
-def test_interpolations(cfg: DictConfig, key: str, expected: Any) -> None:
+def test_interpolations(cfg: Dict[str, Any], key: str, expected: Any) -> None:
     c = OmegaConf.create(cfg)
     assert OmegaConf.select(c, key) == expected
 
 
+@pytest.mark.parametrize(  # type: ignore
+    "cfg,expected_dict",
+    [
+        pytest.param(
+            """
+            a: 1
+            b: a
+            c: ${${b}}
+            """,
+            {"c": 1},
+            id="basic nesting",
```

Comment:
Please avoid special characters in ids, no spaces, no ().
(While this works, it confuses other tools like Pycharm).

Reply:
Done in f25f17f